src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVService/CVObjectDetectionRequest.cs

```diff
@@ -36,8 +36,8 @@ public class CVObjectDetectionRequest
 
         public CVObjectDetectionRequest(CVImageBinaryData screenshot,
                                         [CanBeNull] string textQuery,
-                                        [CanBeNull] CVImageBinaryData imageQuery,
-                                        RectInt? withinRect,
+                                        [CanBeNull] string imageQuery,
+                                        [CanBeNull] RectInt? withinRect,
```

Comment:
[CanBeNull] seems weird/un-necessary on a nullable type

Reply:
I like the visual consistency of things that are can be null but yeah it is redundant. Removed